dotCMS/src/integration-test/java/com/dotcms/datagen/FieldDataGen.java

```diff
@@ -26,13 +24,29 @@
     private boolean unique = Boolean.FALSE;
     private String contentTypeId;
     private String relationType;
+    private String id;
```

Comment:
Codacy found an issue: [Field id has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6326102)